packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts

```diff
@@ -377,12 +377,12 @@ $panel-grid-placeholder-bg: lighten(${theme.palette.blue95}, 30%);
 $panel-grid-placeholder-shadow: 0 0 4px ${theme.palette.blue95};
 
 // logs
-$logs-color-unkown: $gray-5;
+$logs-color-unknown: $gray-5;
```

Comment:
Need to make sure we update references to this.

Reply:
Couldn't find a place where this is used. It should probably be removed.